src/events/events.resolver.ts

```diff
@@ -54,11 +50,11 @@ export class EventsResolver {
       sort,
       query,
       state,
-      userId,
+      null,
       distance,
       latitude,
       longitude,
-      user,
+      userId,
```

Comment:
Jeśli jest tyle argumentów to dobrze przekazać je w postaci obiektu. Jest to dużo czytelniejsze i łatwiejsze w rozbudowanie oraz w definiowaniu domyślnych.